_episodes/003-rust-1.37.0.md

```diff
@@ -10,7 +10,7 @@ length: "26728938" #filesize in byte
 layout: episode
 ---
 
-We review the new features in the Rust 1.37 release and give shout-outs to all the volunteers who have helped make Rustacean Station so far.
+We review the new features in [the Rust 1.37 release](https://blog.rust-lang.org/2019/07/04/Rust-1.36.0.html) and give shout-outs to all the volunteers who have helped make Rustacean Station so far.
```

Comment:
This link doesn't look right?
The link should be https://blog.rust-lang.org/2019/08/15/Rust-1.37.0.html

Reply:
Fixed.